blog/docs/articles/knative-1.0.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Knative 1.0 is out!"
+linkTitle: "Knative 1.0 is out!"
+date: 2021-11-02
+description: "Details on the 1.0 release of Knative"
+type: "blog"
+---
+
+![1.0 Banner Image](/blog/images/1.0Banner.jpg)
+
+***Authors: Carlos Santana (IBM), Omer Bensaadon (VMware), Maria Cruz (Google)***
```

Comment:
```suggestion
***Authors: [Carlos Santana](https://twitter.com/csantanapr) (IBM), [Omer Bensaadon](https://twitter.com/omer_bensaadon) (VMware), [Maria Cruz](https://twitter.com/marianarra_) (Google)***
```

link hanldes

Reply:
Feel free to consider if my contributions are worth a byline.